certified-connectors/PartnerCenterReferrals/apiDefinition.swagger.json

```diff
@@ -86,8 +86,8 @@
             },
```

Comment:
@saferoombiz you are updating another connector in the same pull request. Please remove this change from this PR.